.github/workflows/sync_release-manifest.yml

```diff
@@ -48,3 +48,31 @@ jobs:
           git add .
           git commit -am "${{ github.event.inputs.version }}"
           git push
+
+      - name: Dispatch update-helper update
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
+          # TODO(Rugvip): Remove the create-app dispatch once we've been on the release version for a while
+          script: |
```

Comment:
Upgrade-helper dispatch is moved over here now. Opted for an in-line script to avoid polluting the scripts folder too much